sf-fx-runtime-java-runtime/src/main/java/com/salesforce/functions/jvm/runtime/commands/AbstractDetectorCommandImpl.java

```diff
@@ -94,6 +97,26 @@ public final Integer call() throws Exception {
     return handle(project, functions);
   }
 
+  private static boolean isVersionUnsupported(String salesforceApiVersion) {
+    boolean versionIsUnsupported = true;
+    return parseMajor(salesforceApiVersion)
+        .map(
+            targetVersion ->
+                parseMajor(DEFAULT_SALESFORCE_API_VERSION)
+                    .map(minimumVersion -> targetVersion < minimumVersion)
```

Comment:
Using `flatMap` here will remove the need for one of the two `orElse`s later in the code and IMO makes the intent (same default value for both cases) clearer.